File: Data Structures & Algorithms/palindromic-substrings/submission-2.cpp
class Solution {
public:
    string s;
    int count;
    void expand(int left,int right){
        int n=s.size();
        while(left>=0&&right<n&&s[left]==s[right]){
            count++;
            left--;
            right++;
        }
    }
    int countSubstrings(string s) {
        this->s=s;
        this->count=0;
        int n=s.size();
        for (int i=0;i<n;i++) {
            expand(i,i);
            expand(i,i+1);
        }
        return count;
    }
};

File: Data Structures & Algorithms/binary-tree-postorder-traversal/submission-2.java
class Solution{
    public List<Integer> postorderTraversal(TreeNode root){
        Stack<TreeNode> st=new Stack<>();
        List<Integer> result=new ArrayList<>();
        if(root!=null) st.push(root);
        while(!st.isEmpty()){
            TreeNode node=st.pop();
            result.add(node.val);
            if(node.left!=null) st.push(node.left);
            if(node.right!=null) st.push(node.right);
        }
        Collections.reverse(result);
        return result;
    }
}